src/averageAges.js

```diff
@@ -55,7 +73,26 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const mothers = people.filter(person => person.sex === 'f')
```

Comment:
Here first filter is redundant, because then you have a check `person.mother === mother.name` , so someone with sex 'm', can't be present in `person.mother`